benefits/enrollment/views.py

```diff
@@ -36,9 +36,7 @@ def _index(request):
             viewmodels.Button.primary(
                 text=_("enrollment.buttons.payment_partner"), id=tokenize_button, url=f"#{tokenize_button}"
             ),
-            viewmodels.Button.link(
-                classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=reverse("core:payment_options")
-            ),
+            viewmodels.Button.link(classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=reverse("core:help")),
```

Comment:
Is there a way to link get a link to the `#payment-options` header with `reverse()`?

Reply:
I didn't see anything about doing it with `reverse`, but we can append the anchor to the string returned from `reverse`. I tried out this snippet:
```python
     viewmodels.Button.link(
           classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=f"{help_page}#payment-options"
     ),
```
We get the anchor!
![image](https://user-images.githubusercontent.com/25497886/162536458-3fd376f8-f5f5-45d0-a97c-614936e7ed42.png)

Weird Firefox thing I'm seeing though is that clicking the link will use the right URL, but the browser doesn't go to the anchor. If I refresh the page though, then it goes to the right place on the page. 🤔 Chrome doesn't have this weird issue.

Reply:
@machikoyasuda tested on her Mac with Firefox, and it worked! 🤷